temporalcli/internal/printer/printer_test.go

```diff
@@ -140,3 +143,29 @@ func TestPrinter_JSONList(t *testing.T) {
 	p.EndList()
 	require.Equal(t, "", buf.String())
 }
+
+func TestPrinter_NoPanicIfNoStdout(t *testing.T) {
+	fmt.Println("$$$ TEST v30")
+
+	goPath, err := exec.LookPath("go")
+	if err != nil {
+		t.Fatalf("Error finding go executable: %v", err)
+	}
+	p, err := os.StartProcess(
+		goPath,
+		[]string{"go", "run", "./test/main.go"},
+		&os.ProcAttr{
+			Files: []*os.File{os.Stdin, nil, os.Stderr},
```

Comment:
So the CLI doesn't need changes to support this in non-Windows? Do we need something for Windows?

Reply:
That was fixed in [Go 1.22](https://github.com/golang/go/issues/60641)... But only for unix targets. The issue is technically still possible on Windows, but we haven't seen any practical case of this, so I'll just ignore Windows.